### PR TITLE
Downgrade bash to fix solaris issue

### PR DESCRIPTION
**Pull request by poorndm**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Chore (non-breaking change that does not add functionality or fix an issue)

## Checklist:

- [x] I have read the **CONTRIBUTING** document.
- [ ] I have run the pre-merge tests locally and they pass.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] If `Gemfile.lock` has changed, I have used `--conservative` to do it and included the full output in the Description above.
- [ ] All new and existing tests passed.
- [x] All commits have been signed-off for [the Developer Certificate of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

### CODE REVIEW

**Review by anandhapadmanabhan — APPROVED** [review]

**Comment by poorndm** [comment]
https://buildkite.com/chef-canary/chef-omnibus-toolchain-main-omnibus-adhoc-canary/builds/75 - failing on el-6

**Comment by poorndm** [comment]
> @poorndm Request you to get the PR approved and merged as soon as possible. We will follow up with chef 18 changes after that.

@rahulgoel1 While the build is green on Solaris, it seems to be failing on other platforms. Ensuring a successful build across all platforms is crucial for a stable release of the omnibus-toolchain version. We are working on addressing the issues on the other platforms.

File: Gemfile [hunk 1]
```diff
@@ -3,6 +3,7 @@ source 'https://rubygems.org'
 gem "omnibus", github: ENV.fetch("OMNIBUS_GITHUB_REPO", "chef/omnibus"), branch: ENV.fetch("OMNIBUS_GITHUB_BRANCH", "main")
 gem "omnibus-software", github: ENV.fetch("OMNIBUS_SOFTWARE_GITHUB_REPO", "chef/omnibus-software"), branch: ENV.fetch("OMNIBUS_SOFTWARE_GITHUB_BRANCH", "main")
 gem 'artifactory'
+gem 'unf_ext', '0.0.8'
```
Comment (tpowell-progress) [inline Gemfile:6]: We want the latest `'0.0.8'` which is `'0.0.8.2'` and we should also do pessimistic lock `'~> 0.0.8.2'`

**Review by tpowell-progress — CHANGES_REQUESTED** [review]

File: Gemfile (hunk 1 above)
Comment (tpowell-progress) [reply Gemfile:6]: See diff for [v0.0.8 vs v0.0.8.2](https://github.com/knu/ruby-unf_ext/compare/v0.0.8...v0.0.8.2)

**Comment by poorndm** [comment]
Duplicate of  https://github.com/chef/omnibus-toolchain/pull/271

**Pull request closed without merging**